sdks/unreal/Agones/README.md

```diff
@@ -0,0 +1,65 @@
+# Agones UE4 Plugin
+
+The Agones [plugin](https://docs.unrealengine.com/en-US/Programming/Plugins/index.html)
+for UE4 communicates lifecycle events to the Agones sidecar.
+
+The plugin requires UE4 4.23 or later.
+
+## Installation
+
+Copy the *Agones* folder containing this README to one of the UE4 [Plugin
+Folders](https://docs.unrealengine.com/en-US/Programming/Plugins/#pluginfolders),
+typically the *Plugins* folder at your project root.
+

```

Comment:
It feels like the above should all be part of https://agones.dev/site/docs/guides/client-sdks/unreal/

User documentation should be on the website, not on .md files inside the git repo.

Reply:
I agree, when half the docs are in the repo and half on the website it can be confusing/hidden. Should we move it to `site/content/en/docs/Guides/Client%20SDKs/unreal.md` ?

Reply:
There were already installation instructions in the docs, so I ended up deleting the instructions in the unreal README and linking to the docs instead.